assets/src/components/lateView.tsx

```diff
@@ -86,10 +84,9 @@ const LateView = (): ReactElement<HTMLElement> => {
       <div className="m-late-view__title">Late View</div>
       <div className="m-late-view__panels">
         <div className="m-late-view__panel m-late-view__missing-logons">
-          <h2 className="m-late-view__panel-header m-late-view__missing-logons-panel-header">
-            {userXIcon("m-late-view__panel-header-icon")}
+          <div className="m-late-view__panel-header m-late-view__missing-logons-panel-header">
```

Comment:
**question:** is the move from using an `h2` to a `div` just to avoid some styling we apply to `h2` elements elsewhere? I'd kind of rather keep the semantically-correct element here if possible.

Reply:
IIRC yes but you're right, it would be better to keep the semantics clean.